source/features/bugs-tab.tsx

```diff
@@ -18,11 +23,13 @@ const countBugs = cache.function(async (): Promise<number> => {
 	`);
 
 	return repository.bugs.totalCount;
-}, {
-	maxAge: 1 / 24 / 2, // Stale after half an hour
-	staleWhileRevalidate: 4,
-	cacheKey: (): string => __featureName__ + ':' + getRepoURL()
-});
+}, cacheOptions);
+
+function cleanForComparison(searchQuery: SearchQuery): SearchQuery {
+	const newQuery = searchQuery.clone();
+	newQuery.remove('sort:updated-desc', 'is:open', 'is:closed');
```

Comment:
Do we need to remove @me and author:

Reply:
Only these queries don’t change the numbers on the page. `author` does